packages/core-db/test/app/queued-persisted-processor.spec.ts

```diff
@@ -131,6 +162,84 @@ describe('Queued Persisted Processor', () => {
         `Incorrect item processed!`
       )
     })
+
+    it('retries processing item if handleNextItemThrows', async () => {
+      const first = 'Number 0!'
+      const second = 'Number 1!'
+      await processor.add(0, first)
+      await processor.add(1, second)
+      await sleep(10)
+      processor.handledQueue.length.should.equal(
+        1,
+        `Incorrect number processed!`
+      )
+      processor.handledQueue[0].should.equal(first, `Incorrect item processed!`)
+
+      processor.throwOnceHandlingNextItem = true
+      await processor.markProcessed(0)
+      await sleep(10)
+
+      processor.handledQueue.length.should.equal(
+        1,
+        `There should still only be one item processed! Should fail and retry after 100 millis`
```

Comment:
How likely do you think this will always be 100 millis? If it changes, the `sleep(200)` call below will have to change too. I see there is a `retrySleepDelayMillis` but it is private? If you think that number may increase in the future:
```js
await sleep(processor.retrySleepDelayMillis + 100)
```

If `retrySleepDelayMillis` isn't public, then it could be turned into a constant at the top of the test and passed into the constructor for the `processor` and also used here.

Reply:
Good call. It's configured to be 100 in the constructor in the test [here](https://github.com/ethereum-optimism/optimism-monorepo/pull/224/files#diff-c2017fc68645b7b87f65744f82ef9063R83), so definitely something we can control. To your point, made it a const so we can reference it.